dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -163,10 +188,6 @@ class SentryStackFrame {
       json['in_app'] = inApp;
     }
 
-    if (package != null) {
-      json['package'] = package;
-    }
```

Comment:
why this is removed? `package` is nullable

Reply:
It was assiged twice in the methods.